client/docker/importer/volume/config_instant.json

```diff
@@ -0,0 +1,79 @@
+{
+  "auth": {
+    "secret": "3084e343-71bc-4247-86e4-ea210af89c28",
+    "tokenDuration": 5400
+  },
+  "app": {
+    "port": 3000,
+    "installed": false
+  },
+  "mediator": {
+    "api": {
+      "username": "root@openhim.org",
+      "password": "instant101",
+      "apiURL": "https://openhim-core:8080",
+      "trustSelfSigned": true,
+      "urn": ""
+    },
+    "register": true
+  },
+  "fhirServer": {
+    "baseURL": "http://hapi-fhir:8080/fhir",
+    "username": "hapi",
+    "password": "hapi"
+  },
+  "elastic": {
+    "server": "http://es:9200",
+    "username": "",
+    "password": "",
+    "max_compilations_rate": "10000/1m",
+    "index": "patients"
+  },
+  "codes": {
+    "goldenRecord": "5c827da5-4858-4f3d-a50c-62ece001efea"
+  },
+  "structureDefinition": {
+    "uri": "http://openclientregistry.org/fhir",
+    "reportRelationship": "patientreport"
+  },
+  "matching": {
+    "tool": "elasticsearch"
+  },
+  "clients": [
+    {
+      "id": "openmrs",
+      "displayName": "OpenMRS"
+    },
+    {
+      "id": "lims",
+      "displayName": "Lab Info Management System"
+    }
+  ],
```

Comment:
@citizenrich Should we remove these clients? They don't seem related.
Do they need to be replaced with an Instant client?

Reply:
@ashaban Why do we enumerate clients in the config file?